zookeeper-server/src/test/java/org/apache/zookeeper/test/ClientSSLTest.java

```diff
@@ -84,6 +84,18 @@ public void testClientServerUnifiedPort() throws Exception {
         testClientServerSSL(false);
     }
 
+    @Test
+    public void testClientServerUnifiedPortWithCnxnClassName() throws Exception {
+        System.setProperty(ZKClientConfig.ZOOKEEPER_CLIENT_CNXN_SOCKET, "ClientCnxnSocketNIO");
+        testClientServerSSL(false);
+    }
+
+    @Test
+    public void testClientServerSSLWithCnxnClassName() throws Exception {
+        System.setProperty(ZKClientConfig.ZOOKEEPER_CLIENT_CNXN_SOCKET, "ClientCnxnSocketNetty");
+        testClientServerSSL(true);
```

Comment:
please use System.clearProperty in a finally block in order to clean up the environment

Reply:
the class has `@AfterEach public void teardown()`, so dont need System.clearProperty again?

Reply:
you are right I missed it.